### ISSUE TITLE
Accessibility for Abide (form validation) please

### ISSUE THREAD

**Issue by udf2457** [zurb/foundation-sites#10552]
Hi,

It does not appear that you have implemented any WCAG compliant accessibility for Abide (e.g. setting _aria-invalid_).

**Comment by ncoden** [comment]
See #10699. It should be released in `v6.4.4`.
